tests/unit/test_crud.py

```diff
@@ -186,6 +186,64 @@ def test_get_contact_by_email_id_miss(dbsession):
     assert contact is None
 
 
+def test_get_bulk_contacts_mofo_relevant_false(
+    dbsession, example_contact, maximal_contact, minimal_contact
+):
+    contact_list = [example_contact, maximal_contact, minimal_contact]
+    sorted_list = sorted(
+        contact_list,
+        key=lambda contact: (contact.email.update_timestamp, contact.email.email_id),
+    )
+    mofo_relevant_flag = False
+
+    first_contact = sorted_list[0]
+    after_start = first_contact.email.update_timestamp - timedelta(hours=12)
+    last_contact = sorted_list[-1]
+    last_contact_timestamp = last_contact.email.update_timestamp
+    end_time = last_contact_timestamp + timedelta(hours=12)
+
+    with StatementWatcher(dbsession.connection()) as watcher:
+        bulk_contact_list = get_bulk_contacts(
+            dbsession,
+            start_time=after_start,
+            end_time=end_time,
+            limit=10,
+            mofo_relevant=mofo_relevant_flag,
+        )
+    assert watcher.count == 1
+    assert len(bulk_contact_list) == 0
```

Comment:
This surprised me, I was expecting two results. I think I understand why - there's no row in the mofo table, so the query skips those, even though if you fetch those contacts it will say ``mofo_relevant = false``.

I think either the code should also include contacts with no mofo row, or we should forbid setting the value to false, so that it can only be used to select mofo relevant records and not irrelevant ones

Reply:
I agree, I think?
`mofo_relevant=True` should return all records with that flag set to true in the mofo table, while `mofo_relevant=False` should return all records where (that flag is false OR flag doesn't exist)

Reply:
`mofo_relevant=None` imo should also be a possibility, but without the added filtration.

Reply:
Does ``mofo_relevant=None`` get set by ``/updates?mofo_relevant=`` (no value)? If so, then yes, that should act the same as if you called ``/updates`` without the new parameter.

Reply:
no value... I think we'll need to make a minor shift to the api to allow for "" blank string instead of bools again. But yeah let me make that change